src/engine/ruleset/integrations/apache-http/decoders/apache-error.yml

```diff
@@ -0,0 +1,67 @@
+name: decoder/apache-error/0
+
+sources:
+  - decoder/integrations/0
+
+metadata:
+  module: apache-http
+  title: Apache HTTP Server error logs decoder
+  description: Decoder for Apache HTTP Server error logs.
+  compatibility: The Apache datasets were tested with Apache 2.4.12 and 2.4.46 and are expected to work with all versions >= 2.2.31 and >= 2.4.16 (independent from operating system).
+  versions:
+    - "2.4.12"
+    - "2.4.46"
+  author:
+    name: Wazuh Inc.
+    email: info@wazuh.com
+    date: 2022-11-15
+  references:
+    - https://httpd.apache.org/docs/2.4/logs.html
+    - https://httpd.apache.org/docs/2.4/custom-error.html
+
+check:
+  - event.original: +starts_with/[
```

Comment:
Leave a comment with the `check` that would be used with the configuration labels. This way, if the integration model we want is applied, the change can be automatic.

```
#TODO: Once the events arrive tagged, uncomment these lines below and remove the above `event.original`
  # - event.module: apache-http
  # - event.dataset: apache-error
  ```

Reply:
done